Cargo.toml

```diff
@@ -25,14 +25,15 @@ git2 = "0.13.17"
 git-url-parse = "0.3.1"
 heck = "0.3.2"
 humantime = "2.1.0"
+os_info = "3.0"
 prettytable-rs = "0.8.0"
 serde = "1.0"
 serde_json = "1.0"
 structopt = "0.3.21"
 tracing = "0.1.22"
 regex = "1"
 url = "2.2.0"
-os_info = "3.0"
+webbrowser = "0.5.5"
```

Comment:
You probably looked at how `cargo-doc` does it already, _but in case you haven't_, here is a link to how they just use `std::process::Command` to [run docs in browser](https://github.com/rust-lang/cargo/blob/master/src/cargo/ops/cargo_doc.rs#L101-L121)

Reply:
it looks like they use the `opener` crate if the `BROWSER` env var isn't present. The `webbrowser` crate I've selected does the `BROWSER` env var handling AND the cross platform fallback provided by the `opener` crate